### PR TITLE
feat: rename to Derailed

### PR DESCRIPTION
**Pull request by teaishealthy**

### CODE REVIEW

File: .github/workflows/build-cli.yml [hunk 1]
```diff
@@ -88,21 +88,21 @@ jobs:
...
+          chmod +x derailed
+          cp -p derailed .debpkg/usr/bin/
+          cp -p derailed .rpmpkg/usr/bin/
       - name: Build DEB
         uses: jiro4989/build-deb-action@v2
         with:
           package: ${{ env.NAME }}
           package_root: .debpkg
-          maintainer: Oliver Wilkes <oliver@eludris.gay>
+          maintainer: Oliver Wilkes <oliver@derailed.gay>
```
Comment (toolifelesstocode) [inline .github/workflows/build-cli.yml:105]: So who's gonna buy this domain

Comment (EnokiUN) [reply .github/workflows/build-cli.yml:105]: skl

**Comment by teaishealthy** [comment]
whats the status on this?

**Comment by EnokiUN** [comment]
> whats the status on this?

We are seriously considering this with all our might on a minute by minute basis! Please be a bit more patient.

**Comment by EnokiUN** [comment]
After a bunch of consideration, we've concluded that doing this would lead to my instantaneous implosion IRL (which I'd personally like to avoid).

Such a shame, it's a really good idea too!

**Pull request closed without merging**